CodeSniffer/CLI.php

```diff
@@ -135,6 +135,14 @@ public function runphpcbf()
         $cliValues['reportFile']   = null;
         $cliValues['reports']      = array();
 
+        if ((isset($cliValues['no-patch']) === false)
+            && ($this->_commandExist('diff') === false && $this->_commandExist('patch') === false)
```

Comment:
Maybe we should use `||` instead, because if user have diff installed, but patch isn't installed this ends up in same error as before this PR.